cabot/cabotapp/tests/tests_basic.py

```diff
@@ -396,6 +404,10 @@ def test_duplicate_statuscheck(self):
         self.assertEqual(new.endpoint, old.endpoint)
         self.assertEqual(new.status_code, old.status_code)
 
+    @patch('cabot.cabotapp.tasks.run_status_check', fake_run_status_check)
+    def test_run_all(self):
+        tasks.run_all_checks()
```

Comment:
Fine with only this as long as run_all_checks calls _classify_status_check, otherwise would add a test for _classify_status_check